src/www/ui/api/Helper/UploadHelper.php

```diff
@@ -627,6 +628,31 @@ public function getUploadCopyrightList($uploadId)
     return $responseList;
   }
 
+  /**
+    *  Get the clearing status for files within an upload
+     * @param DbManager $dbManager DbManager object
+     * @param integer $uploadTreeTableId upload tree id of the file
+     * @param integer $groupId groupId of the user
+     * @return String String containing the Clearing status message
+*/
+
+  public function fetchClearingStatus($dbManager, $uploadTreeTableId, $groupId)
+  {
+    $clearingStatus = [];
+    $clearingStatusMsg = array("WIP",null,null,"TO_BE_DISCUSSED","IRRELEVANT","IDENTIFIED","DO_NOT_USE","NON_FUNCTIONAL");
```

Comment:
Please create object of class [`Fossology\Lib\Data\DecisionTypes`](https://github.com/fossology/fossology/blob/43924f05229e03bfa1c86eb285127f9f0b93d409/src/lib/php/Data/DecisionTypes.php#L10) instead.

Reply:
Resolved